src/openms/include/OpenMS/SYSTEM/RWrapper.h

```diff
@@ -74,12 +76,12 @@ namespace OpenMS
     static String findScript(const String& script_file, bool verbose = true);
 
     /**
-      @brief Check for presence of 'Rscript' in system path.
+      @brief Check for presence of 'Rscript'.
 
       @param verbose Print failure information?
       @return Success status
     */
-    static bool findR(bool verbose = true);
+    static bool findR(const QString& executable = QString("Rscript"), bool verbose = true);
```

Comment:
`OpenMS::String` would be more consistent here. Would also remove the dependency on QT headers here.